src/interface/src/app/shared/file-uploader/file-uploader.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FileUploaderComponent } from './file-uploader.component';
+
+describe('FileUploaderComponent', () => {
+  let component: FileUploaderComponent;
+  let fixture: ComponentFixture<FileUploaderComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ FileUploaderComponent ]
+    })
+    .compileComponents();
+
+    fixture = TestBed.createComponent(FileUploaderComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
should this contain a test for onFileUploaded?

Reply:
aha almost got away with it.. ;) done!